exercises/space-age/.meta/generator/test_template.erb

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env ruby
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative '<%= exercise_name %>'
+
+# Common test data version: <%= abbreviated_commit_hash %>
+class <%= exercise_name_camel %>Test < Minitest::Test
+  DELTA = 0.01
```

Comment:
I've though about what this comment should say, and I'm kind of embarrassed to say that I don't really know what delta means in this case. Besides the basic mathematics definition of "change in a given quantity", I'm kind of stumped.